web/app/frontend/src/components/Router.tsx

```diff
@@ -73,8 +74,8 @@ export default function Router() {
                                 title: routes.addRecord.title
                             }}
                         >
-                            {(props) =>
-                                <AddRecordScreen {...props} state={state} dispatch={dispatch}/>
+                            {({navigation, route}) =>
+                                <AddRecordScreenContainer navigation={navigation} route={route as AddRecordScreenContainerProps['route']} state={state} dispatch={dispatch}/>
```

Comment:
can the linting be set up so each property is on its own line?

Reply:
just tested it, it already does that. I just didn't run the fix on the whole package because it would make a massive PR. We should probably submit a dedicated PR for that.
I'll run it for the files in this PR though